File: June/ex17.py
#!/usr/bin/python
from sys import argv
from os.path import exists

#script, from_file, to_file = argv

open(argv[2],'w').write(open(argv[1],'r').read())
#print "Copeing from %s to %s" %(from_file, to_file)
#
#indata = open(from_file).read()
#
#print "The input file is %d bytes log" % len(indata)
#
#print "Does the output file exist? %r" % exists(to_file)
#print "Ready, hit RETURN to continue, CTRL-C to abort."
#raw_input()
#
#output = open(to_file, 'w')
#output.write(indata)
#
#print "Alright, all done."
#
#output.close()

File: Learn_Ruby_The_Hard_Way/ex17.rb
from_file, to_file = ARGV
script = $0

puts "Copying from #{from_file} to #{to_file}"

indata = File.open(from_file).read()

puts "This input file is #{indata.length} bytes long"

puts "Does the output file exist? #{File.exist? to_file}"
puts "Ready, hit RETURN to continue, CTRL-C to abort."
STDIN.gets

output = File.open(to_file, 'w')
output.write(indata)

puts "Alright, all done."

output.close()
